source/configuring_domain/03_managing_tracking/third_party_analytics.rst

```diff
@@ -4,56 +4,45 @@ Third-party analytics
 The monitoring tools provided by Opendatasoft focus on API calls and usages.
 They are useful to gather precise information on how data is consumed (see :doc:`Analyzing domain usage </managing_domain/03_analyzing_domain_usage/index>`). But it can be limiting when trying to measure more broadly the audience of a portal: referral sources, visitors' geographical origin, time spent on the portal, etc.
 
-Third-party services, specialized in measuring audience, can give additional analytics such as those mentioned above. Opendatasoft supports integration with Google Analytics, and XiTi and SmartTag by AT Internet.
+Third-party services, specialized in measuring audience, can give additional analytics such as those mentioned above. Opendatasoft supports integration with Google Analytics, XiTi and SmartTag by AT Internet, and Matomo.
```

Comment:
"Google Analytics, XiTi and SmartTag by AT Internet, and Matomo" => 
"Google Analytics, XiTi, SmartTag by AT Internet, and Matomo"

Reply:
[Fixed](https://github.com/opendatasoft/ods-documentation/pull/286/files#diff-c6e440d6fd65214262f0083fffdade1a620fde20b4a04f2b43186c669a247b11R7)